SparkPost/2web2ui
# FE-725 - Focus is now managed properly following route changes

[FE-725](https://jira.int.messagesystems.com/browse/FE-725)

### What Changed
- Adds new `<RouteFocusHandler/>` component that renders immediately inside router context for both private and public routes
- Adds new [aria-live region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) to communicate page changes to screen readers

### How To Test
- Following a route change, in the dev console, run `document.activeElement` to verify that the focused element is the `<body>`
- Go to a page like `/report/summary` and update query params - verify that focus does _not_ move to the `<body>`

### To Do
